crates/autopilot/src/database/onchain_order_events.rs

```diff
@@ -381,6 +390,7 @@ async fn get_quote(
     })
 }
 
+#[allow(clippy::too_many_arguments)]
```

Comment:
It's a sin, but I think this is a lesser problem. Really what this points to is a lack of proper abstractions.